Libraries/LibWeb/SVG/SVGFilterElement.cpp

```diff
@@ -170,6 +173,87 @@ Optional<Gfx::Filter> SVGFilterElement::gfx_filter(Layout::NodeWithStyle const&
 
             root_filter = Gfx::Filter::blur(radius_x, radius_y, input);
             update_result_map(*blur_primitive);
+        } else if (auto* colormatrix_primitive = as_if<SVGFEColorMatrixElement>(node)) {
+            auto in_attr = colormatrix_primitive->in1()->base_val();
+            auto input = resolve_input_filter(in_attr);
+
+            auto type_value = colormatrix_primitive->attribute(AttributeNames::type).value_or(String {});
+            auto values_value = colormatrix_primitive->attribute(AttributeNames::values).value_or(String {});
+
+            auto equals_ci = [](StringView a, StringView b) {
+                return AK::StringUtils::equals_ignoring_ascii_case(a, b);
+            };
```

Comment:
This helper seems a bit useless, you can just call `type_value.equals_ignoring_ascii_case(StringView)`?